index.bs

```diff
@@ -70,6 +70,11 @@ urlPrefix: https://www.w3.org/TR/payment-request/; spec: payment-request
 
 <h2 id="basic-principles">Principles behind design of Web APIs</h2>
 
+The Design Principles are directly informed by the ethical framework set out in the
+Ethical Web Principles [[ETHICAL-WEB]]. These principles provide concrete practical
+advice in response to the higher level ethical responsibilities that come with
+developing the Web platform.
+
```

Comment:
We introduced more aggressive line breaking during a rewrite of this document, would it be possible to break the lines following that convention?

(The rationale was to make diffs easier to follow, but also to make it easier to read)